cns/NetworkContainerContract.go

```diff
@@ -220,3 +227,29 @@ type UnpublishNetworkContainerResponse struct {
 	UnpublishStatusCode   int
 	UnpublishResponseBody []byte
 }
+
+// Validate - Validates network container request policies
+func (networkContainerRequestPolicy *NetworkContainerRequestPolicies) Validate() error {
+
+	// validate ACL policy
+	if strings.EqualFold(networkContainerRequestPolicy.Type, "ACLPolicy") {
+
+		var requestedAclPolicy hcn.AclPolicySetting
+
+		if err := json.Unmarshal(networkContainerRequestPolicy.Settings, &requestedAclPolicy); err != nil {
+
+			return fmt.Errorf("ACL policy failed to pass validation with error: %+v ", err)
+		}
+
+		if requestedAclPolicy.Action == "" {
+			return errors.New("Action field cannot be empty in ACL Policy")
+		}
+
+		if requestedAclPolicy.Priority == 0 {
+			return errors.New("Priority field cannot be empty in ACL Policy")
```

Comment:
I see you are creating error using fmt.Errorf on line 241 and errors.New in other places. Can you use either one of those in all the places? fmt.Errorf is what we have used (in most places) in the code

Reply:
Corrected that.